MultiplayerMod/Platform/Steam/Network/Messaging/Surrogates/CarePackageInstanceDataSurrogate.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Runtime.Serialization;
+
+namespace MultiplayerMod.Platform.Steam.Network.Messaging.Surrogates;
+
+public class CarePackageInstanceDataSurrogate : ISerializationSurrogate, ISurrogateType {
+
+    public Type Type => typeof(CarePackageContainer.CarePackageInstanceData);
+
+    public void GetObjectData(object obj, SerializationInfo info, StreamingContext context) {
+        var node = (CarePackageContainer.CarePackageInstanceData) obj;
+        info.AddValue("facadeID", node.facadeID);
+        info.AddValue("info.id", node.info.id);
+        info.AddValue("info.quantity", node.info.quantity);
+        info.AddValue("info.facadeID", node.info.facadeID);
```

Comment:
Is this different with `facadeId`?

Reply:
They might be different if info.facadeId is 'SELECTRANDOM' than facadeID will be choosen upon generation and must be sent with generated value instead if `SELECTRANDOM`